SparkPost/support-docs
# WE-277 Add meta tags

**What Changed**
- Adds seo meta tags
- Adds new favicon + social share image

**How to verify**
- Visit the deploy preview or run the site locally
- Visit any momentum doc like `/momentum/4/4-preface`
  - Verify the title, description and other meta tags are correct
- Visit the momentum index page `http://localhost:3000/momentum`
  - Verify the title and description are correct and match the existing momentum site
